ajax/libs/firmin/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "firmin",
+  "filename": "firmin-1.0.0-min.js",
+  "description": "JavaScript animation library based on CSS transforms and transitions.",
```

Comment:
Let's add a `A` in the description, the whole line is `A JavaScript animation library using CSS transforms and transitions.`
(As the same with upstream's description in top of GitHub repo.)
Many thanks